web/src/portico/help.ts

```diff
@@ -6,12 +6,15 @@ import * as common from "../common";
 import * as google_analytics from "./google-analytics";
 import {activate_correct_tab} from "./tabbed-instructions";
 
-function registerCodeSection($codeSection) {
+function registerCodeSection($codeSection: JQuery): void {
     const $li = $codeSection.find("ul.nav li");
     const $blocks = $codeSection.find(".blocks div");
 
     $li.on("click", function () {
         const language = this.dataset.language;
+        if (language === undefined) {
+            throw new Error("Element's data-language attribute must be defined.");
```

Comment:
We can use `minimalistic-assert` in place of `throw new Error` for here and the following checks.

Reply:
Adjusted accordingly. Please check.